fabbwled-backend/src/main/resources/sections/253-288.yaml

```diff
@@ -29,3 +29,12 @@
         - turnTo: 131
       else:
         - turnTo: 537
+- number: 269
+  content:
+    - text: |
+        You find a natural alcove in the wall of a tunnel. Desperately,
+        you squeeze yourself into it, and fade into the shadows. The
+        ratmen come bundling past, whooping and yelling. None of
+        them spot you, and once they have gone you step out, smiling
+        to yourself. Turn to 580.
```

Comment:
```suggestion
        to yourself.
```

Reply:
yeah, but we need to implement some easy sections to show some results.